pkg/nsx/services/ipblocksinfo/ipblocksinfo.go

```diff
@@ -392,11 +417,37 @@ func (s *IPBlocksInfoService) getIPBlockCIDRsFromStore(pathSet sets.Set[string],
 			return nil, fmt.Errorf("failed to get IPBlock %s from NSX", path)
 		}
 		ipblock := obj.(*model.IpAddressBlock)
-		if ipblock.Cidr == nil { //nolint:staticcheck //ipblock.Cidr is deprecated
-			return nil, fmt.Errorf("failed to get CIDR from ipblock %s", path)
+		if ipblock.Cidrs != nil {
```

Comment:
Maybe check len(ipblock.Cidrs) > 0?

Reply:
it should be safe even len(ipblock.Cidrs) ==0 for ipCIDRs = append(ipCIDRs, ipblock.Cidrs...)